packages/react/src/card/card.tsx

```diff
@@ -227,11 +238,16 @@ export const Card = forwardRef<HTMLDivElement, CardProps | CardFocusProps>(
           { "hds-card--color-white": effectiveColor === "white" },
           { "hds-card--color-light-grey-fill": effectiveColor === "light-grey-fill" },
           { "hds-card--color-darker": effectiveColor === "darker" },
+          { "hds-card--imagepos-right": imageRight },
           className as undefined,
         )}
         ref={ref}
       >
-        {children}
+        {variant === "full-width" ? (
+          <div className="hds-card__layoutwrapper">{children}</div>
```

Comment:
Use the `clsx` utility, even for single classnames that does not allow extension. It ensures type-safety. Meaning if the class `hds-card__centerbody` is ever removed/renamed, typescript will complain

Reply:
👍